module.rb

```diff
@@ -0,0 +1,28 @@
+module MyEnumerable
+  def list
+    @list ||= []
+  end
+
+  def all?
+    ary = []
+    list.each do |item|
+      ary << yield(item)
+    end
+    p ary.length == list.length
+  end
+
+  def any?
+    list.each do |item|
+      return p true if yield(item)
+    end
+    p false
+  end
+
+  def filter
+    ary = []
+    list.filter do |item|
+      ary.push(item) if yield item
+    end
+    p ary
+  end
```

Comment:
- [ ] These enumerables should be implemented using the "my_each" method created in the MyList class. For example

  ```
  module MyEnumerable
    def all?
      my_each { |item| return false unless yield item }
      
      true
    end
  
    def any?
      my_each { |item| return true if yield item }
      false
    end
  
    def filter
      filter_array = []
      my_each { |item| filter_array << item if yield item }
  
      filter_array
    end
  end
  ```